roles/cloud-bootstrap/tasks/create-users-projects.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Create bonnyci project
+  os_project:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci-admin group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci-admin
+
+- name: Add all users to bonnyci group
+  os_user_group:
+    cloud: "{{ cloud }}"
+    group: bonnyci
+    user: "{{ item.username }}"
+  with_items: users
```

Comment:
With ansible 2.x, the argument to loops needs to be templated.

`with_items: {{ users }}`